src/wrappers/themis/wasm/test/test.js

```diff
@@ -20,7 +20,7 @@ const ThemisErrorCode = themis.ThemisErrorCode
 
 function expectError(errorCode) {
     return function(error) {
-        if (error instanceof themis.ThemisError) {
+        if (error.errorCode) {
```

Comment:
instanceof is also not supported in ES5. Currently tests run with ES5 to catch comparability issues.